sdc/tests/test_dataframe.py

```diff
@@ -544,6 +544,50 @@ def test_impl(n):
         self.assertEqual(count_array_REPs(), 0)
         self.assertEqual(count_parfor_REPs(), 0)
 
+    @skip_sdc_jit
+    def test_index_attribute(self):
```

Comment:
I propose to add generic method to test `df.index`, something like that:
```
def _test_df_index(self, df):
    def test_impl(df):
        return df.index

    sdc_func = self.jit(test_impl)
    np.testing.assert_array_equal(sdc_func(df), test_impl(df))
```
then use it in 3 places, e.g.:
```
@skip_sdc_jit
def test_df_index_attribute_empty(self):
    n = 5
    A = np.ones(n)
    B = np.random.ranf(n)
    df = pd.DataFrame({'A': A, 'B': B})

    self._test_df_index(df)
```